dactyl-config.yml

```
@@ -4100,6 +4100,13 @@ pages:
         targets:
             - en
 
+    -   name: XRPL Jobs
+        html: http://jobs.xrpl.org/
```

Let's upgrade this to HTTPS as a good practice.

```suggestion
        html: https://jobs.xrpl.org/
```

---

Agreed. I should have added that